config/routes.rb

```diff
@@ -12,10 +12,24 @@
 
   mount SwaggerUiEngine::Engine, at: '/api'
 
-  def add_swagger_route http_method, path, opts = {}
+  def swagger_base_path
+    @swagger_base_path ||= YAML.load_file(Rails.root.join("public/doc/swagger-2.yaml"))["basePath"]
+  end
+
+  def full_path(path)
```

Comment:
@Fryguy we use mustache templates to generate the routes so this code should be in the template file
https://github.com/ManageIQ/insights-api-service_portal/blob/2b5101d65db9bf28ef5638f073f1dbd7c57a50f3/swagger-codegen-templates/routes.mustache#L7

Reply:
Oh nice, I didn't know that!  Will do!